README.md

```diff
@@ -236,6 +236,43 @@ to look for module parameters. There are multiple ways how the installer can sel
   * automatically if there is only one scenario available
   * automatically if installer was ran already with scenario selected
 
+### Re-installing with different scenario
+
+Lets assume you have already completed installation with one scenario (e.g. smart-proxy).
+Now you want to reinstall or upgrade with different scenario (e.g. foreman). This is tricky
+situation and may end with unpredictable results so you should double check
+if the scenario and the puppet modules used in it support such kind of change.
+
+Installer tries to prevent unintentional change of a scenario and interrupts when such situation is detected:
+```bash
+  $> foreman-installer -S foreman-installer
```

Comment:
other examples are either with `$` or without anything, please choose one to not introduce even more inconsistency